core/templates/pages/interaction-specs.constants.ajs.ts

```diff
@@ -16,8 +16,7 @@
  * @fileoverview Constant file for the INTERACTION_SPECS constant.
  */
 
-import INTERACTION_SPECS from 'interactions/interaction_specs.json';
+import { InteractionSpecsConstants } from 'pages/interaction-specs.constants';
```

Comment:
@vojtechjelinek We are expecting to use the constants defined in the *.constants.ts file, somehow the lint check was not working for this file, I've fixed the issue and had to change this file.

Reply:
LGTM!